packages/patternfly-4/react-core/src/components/AboutModal/AboutModal.js

```diff
@@ -33,6 +33,8 @@ const propTypes = {
     }
     return null;
   },
+  /** prevents AboutModal from rendering content inside AboutModalContentBox; allows for more flexible layouts */
```

Comment:
Maybe update the description to sentence case.   I don't know  that AboutModalContentBox is meaningful to the consumer.

Reply:
Would "Prevents AboutModal from rendering content inside a container" be more meaningful?